sdk/storage/tests.yml

```diff
@@ -1,34 +1,4 @@
 trigger: none
 
 extends:
-  template: /eng/pipelines/templates/stages/archetype-sdk-tests.yml
-  parameters:
-    ServiceDirectory: storage
-    BuildInParallel: true
-    TimeoutInMinutes: 180
-    Location: canadacentral
-    CloudConfig:
-      Preview:
-        SubscriptionConfiguration: $(sub-config-azure-cloud-test-resources-preview)
-      PrivatePreview:
-        SubscriptionConfiguration: $(sub-config-storage-test-resources)
-    Clouds: Preview
-    SupportedClouds: Preview, PrivatePreview
-    MatrixReplace:
-      # Use dedicated storage pool in canadacentral with higher memory capacity
-      - Pool=(.*)-general/$1-storage
-    ${{ if contains(variables['Build.DefinitionName'], 'tests-weekly') }}:
-      MatrixConfigs:
-        - Name: Storage_all_versions
-          Path: sdk/storage/platform-matrix-all-versions.json
-          Selection: sparse
-          GenerateVMJobs: true
-    EnvVars:
-      DYLD_LIBRARY_PATH: /usr/local/opt/openssl@3/lib
-      ${{ if not(contains(variables['Build.DefinitionName'], 'tests-weekly')) }}:
-        AZURE_ONLY_TEST_LATEST_SERVICE_VERSION: true
-    TestSetupSteps:
-    - template: /sdk/storage/tests-install-azurite.yml
-    - bash: brew install openssl@3
-      displayName: (MacOS) Install OpenSSL
-      condition: contains(variables['OSVmImage'], 'mac')
+  template: /test-stages.yml
```

Comment:
Doesn't the / in front say to find this from the root of the repo?

Reply:
it does. good catch